wordpress-mobile/WordPress-iOS
# Connect Jetpack from Notifications with new flow

**Fixes** #8571

This PR builds on https://github.com/wordpress-mobile/WordPress-iOS/pull/8454 and https://github.com/wordpress-mobile/WordPress-iOS/pull/8558 to trigger the new Jetpack connection flow from Notifications.

**To test:**

1. Start with a clean slate.
2. Add a self hosted site: https://jurassic.ninja/create/?nojetpack
3. Go to Notifications and connect that site to a Jetpack account.
4. Check that after that you get notifications :)

@ScoutHarris  sending this one your way so you get some Jetpack insights :)

cc: @koke
